tools/functions.sh

```diff
@@ -6,8 +6,8 @@
 # See LICENSE file for more details.
 ###############################################################
 
-# shellcheck disable=SC2034
-installdir="${rootdir:?}/build/install"
```

Comment:
Actually, this entire PR could be avoided by simply removing this line and moving it to the only script that uses it, run.sh.

Reply:
That's right. I will create a separate PR which does that.

Reply:
Not only this, but also remove scriptdir and rootdir from owncloud_definitions.sh. Any script which may be sourced should not define global variables which may be used by the callers.

Reply:
https://github.com/prplfoundation/prplMesh/pull/1218